internal/publish/config.go

```diff
@@ -61,6 +61,12 @@ type Config struct {
 
 	RevisionKeyCacheDuration time.Duration `env:"REVISION_KEY_CACHE_DURATION, default=1m"`
 
+	// AllowPartialRevisions permits uploading multiple exposure keys with a
+	// revision token where only a subset of the keys are in the token. In that
+	// case, only the incoming exposure keys that match the revision token are
+	// uploaded and the remainder are discarded.
+	AllowPartialRevisions bool `env:"ALLOW_PARTIAL_REVISIONS, default=false"`
```

Comment:
I think this makes more sense as a key server config instead of a per-app config, but we could easily make it per-app too. Curious your thoughts @mikehelmick.

Reply:
I think server level config is fine.